doc/source/basics.rst

```diff
@@ -307,14 +307,13 @@ To evaluate single-element pandas objects in a boolean context, use the method
 
    .. code-block:: python
 
-       >>> if df:                         # noqa: E999
-               ...
+      >>> if df:                         # noqa: E999
```

Comment:
If the `...` is giving problems with the validation I'd replace it by a `pass`, and we can also remove the `noqa` comment then.